File: python/kode program/minuman.py
# minuman.py

from menu import Menu

class Minuman(Menu):
    def __init__(self, nama, harga, kode_menu, deskripsi, ukuran, suhu):
        super().__init__(nama, harga, kode_menu, deskripsi)
        self.ukuran = ukuran  # kecil / sedang / besar
        self.suhu = suhu # Panas / Dingin

    def tampilkan(self):
        # Memanggil metode tampilkan dari parent class dan menambahkan info spesifik
        super().tampilkan()
        print(f"    Info: Ukuran {self.ukuran}, Suhu: {self.suhu}")

File: java/kode program/Minuman.java
// Minuman.java
public class Minuman extends Menu {
    private String ukuran;
    private String suhu;

    public Minuman(String nama, int harga, String kodeMenu, String deskripsi, String ukuran, String suhu) {
        // Memanggil constructor dari superclass (Menu)
        super(nama, harga, kodeMenu, deskripsi);
        this.ukuran = ukuran;
        this.suhu = suhu;
    }

    // Getter untuk atribut spesifik Minuman
    public String getUkuran() { return ukuran; }
    public String getSuhu() { return suhu; }

    @Override
    public void tampilkanDetail() {
        System.out.printf("[%s] %s - Rp%,d\n", this.kodeMenu, this.nama, this.harga);
        System.out.printf("    Deskripsi: %s\n", this.deskripsi);
        System.out.printf("    Info: Ukuran %s, Suhu: %s\n", this.ukuran, this.suhu);
    }
}